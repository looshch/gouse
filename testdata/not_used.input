package p

// Tests if unused variables get fake usages and if every second processing of
// the same file returns it to its exact initial state. Checks both a case with
// a single definition per line and one with multiple assignments per line.
func main() {
	var (
		notUsed0 = false
		used0    bool
	)
	notUsed1, used1 := "", ""
	_, _ = used0, used1
}
