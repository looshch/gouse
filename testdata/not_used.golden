package p

func main() {
	var (
		notUsed0 = false; _ = notUsed0 /* TODO: gouse */
		used0    bool
	)
	notUsed1, used1 := "", ""; _ = notUsed1 /* TODO: gouse */
	_, _ = used0, used1
}
